build.gradle

```diff
@@ -772,6 +775,37 @@ distributions {
     }
   }
 }
+
+ospackage {
+  packageName = distributions.main.baseName
+  version = version
+
+//  release = new Date().format('yyyyMMdd-HHmmss')
+//  arch = NOARCH
+
+  os = LINUX
+
+  maintainer = 'SnappyData, Inc.'
+  packageDescription = productName + ', the Spark Database. Stream - Transact - Analyze - Predict all in one cluster'
+  summary = productName + ' Installer'
+
+  user = 'root'
+  permissionGroup = 'root'
+
+  from(snappyProductDir) {
+//    fileMode = 0664
+    into '/opt/' + distributions.main.baseName
+  }
+}
+
+buildRpm {
+  requires('glibc')
+}
+
+buildDeb {
+  requires('libc6')
+}
```

Comment:
Ideally we can add dependency on "java-1.8.0" for RPM and "java8-sdk" for DEB. These are virtual names and all standard java RPM/DEB (checked both openjdk and Oracle distribution/ubuntu PPAs) use these. However it is quite possible that some users just get the Oracle tar.gz and have it installed in some common place. So for DEB we can add a recommends('java8-sdk') while for RPM there is no equivalent (recent Fedora have something called "weak dependencies" but that is not available in RHEL/Centos AFAIK).